From 1cccee1488144106173d619c97d021b2aeb45a9b Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Wed, 10 Apr 2019 19:42:37 +0200
Subject: [PATCH] board: rename tqma8qx -> tqma8xx

This rename should (hopefully) correct the naming mess:

module is TQMa8Xx (X for the i.MX8 X family, x for CPU variant)

therefore:

- variant with i.MXDX is named TQMa8XD
- variant with i.MXQXP is named TQMa8XD

We rename / move here:

- config names
- file names
- board names
- Kconfig symbols

While at it, improve naming of RAM / CPU and baseboard Kconfig choice

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/dts/Makefile                         |  4 +-
 ...a8xx.dts => fsl-imx8dx-tqma8xd-mba8xx.dts} |  6 +-
 ...xx.dts => fsl-imx8qxp-tqma8xqp-mba8xx.dts} |  6 +-
 ...qma8qxp.dtsi => fsl-imx8qxp-tqma8xqp.dtsi} |  4 +-
 arch/arm/mach-imx/imx8/Kconfig                |  6 +-
 board/tqc/tqma8qx/Kconfig                     | 96 -------------------
 board/tqc/tqma8xx/Kconfig                     | 96 +++++++++++++++++++
 board/tqc/{tqma8qx => tqma8xx}/Makefile       |  5 +-
 .../tqma8xx-mba8xx.c}                         |  2 +
 .../{tqma8qx/tqma8qx.c => tqma8xx/tqma8xx.c}  |  0
 ...defconfig => tqma8xd_mba8xx_mmc_defconfig} |  4 +-
 ...efconfig => tqma8xqp_mba8xx_mmc_defconfig} |  2 +-
 .../{tqma8qx-mba8xx.h => tqma8xx-mba8xx.h}    |  6 +-
 include/configs/{tqma8qx.h => tqma8xx.h}      | 28 +++---
 14 files changed, 133 insertions(+), 132 deletions(-)
 rename arch/arm/dts/{fsl-imx8dx-tqma8dx-mba8xx.dts => fsl-imx8dx-tqma8xd-mba8xx.dts} (88%)
 rename arch/arm/dts/{fsl-imx8qxp-tqma8qxp-mba8xx.dts => fsl-imx8qxp-tqma8xqp-mba8xx.dts} (97%)
 rename arch/arm/dts/{fsl-imx8qxp-tqma8qxp.dtsi => fsl-imx8qxp-tqma8xqp.dtsi} (98%)
 delete mode 100644 board/tqc/tqma8qx/Kconfig
 create mode 100644 board/tqc/tqma8xx/Kconfig
 rename board/tqc/{tqma8qx => tqma8xx}/Makefile (51%)
 rename board/tqc/{tqma8qx/tqma8qx-mba8xx.c => tqma8xx/tqma8xx-mba8xx.c} (99%)
 rename board/tqc/{tqma8qx/tqma8qx.c => tqma8xx/tqma8xx.c} (100%)
 rename configs/{tqma8dx_mba8xx_mmc_defconfig => tqma8xd_mba8xx_mmc_defconfig} (96%)
 rename configs/{tqma8qx_mba8qx_mmc_defconfig => tqma8xqp_mba8xx_mmc_defconfig} (98%)
 rename include/configs/{tqma8qx-mba8xx.h => tqma8xx-mba8xx.h} (86%)
 rename include/configs/{tqma8qx.h => tqma8xx.h} (96%)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 2ea20e2c37..03fbac83fd 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -472,14 +472,14 @@ dtb-$(CONFIG_ARCH_IMX8M) += fsl-imx8mq-evk.dtb \
 		fsl-imx8mq-tqma8mq-mba8mx.dtb
 
 dtb-$(CONFIG_ARCH_IMX8) += fsl-imx8dx-ddr3-arm2.dtb \
-		fsl-imx8dx-tqma8dx-mba8xx.dtb \
+		fsl-imx8dx-tqma8xd-mba8xx.dtb \
 		fsl-imx8qm-ddr4-arm2.dtb \
 		fsl-imx8qm-lpddr4-arm2.dtb \
 		fsl-imx8qm-mek.dtb \
 		fsl-imx8qm-mek-xen.dtb \
 		fsl-imx8qxp-lpddr4-arm2.dtb \
 		fsl-imx8qxp-mek.dtb \
-		fsl-imx8qxp-tqma8qxp-mba8xx.dtb \
+		fsl-imx8qxp-tqma8xqp-mba8xx.dtb \
 		fsl-imx8qxp-tqma8qxps-mb-smarc-2.dtb
 
 dtb-$(CONFIG_RCAR_GEN3) += \
diff --git a/arch/arm/dts/fsl-imx8dx-tqma8dx-mba8xx.dts b/arch/arm/dts/fsl-imx8dx-tqma8xd-mba8xx.dts
similarity index 88%
rename from arch/arm/dts/fsl-imx8dx-tqma8dx-mba8xx.dts
rename to arch/arm/dts/fsl-imx8dx-tqma8xd-mba8xx.dts
index 9db2b42911..4768c3c419 100644
--- a/arch/arm/dts/fsl-imx8dx-tqma8dx-mba8xx.dts
+++ b/arch/arm/dts/fsl-imx8dx-tqma8xd-mba8xx.dts
@@ -12,11 +12,11 @@
  * GNU General Public License for more details.
  */
 
-#include "fsl-imx8qxp-tqma8qxp-mba8xx.dts"
+#include "fsl-imx8qxp-tqma8xqp-mba8xx.dts"
 
 / {
-	model = "TQ Systems i.MX8DX TQMa8DX on MBa8Xx";
-	compatible = "tq,tqma8dx-mba8xx", "tq,tqma8dx", "fsl,imx8qxp";
+	model = "TQ Systems i.MX8DX TQMa8XD on MBa8Xx";
+	compatible = "tq,tqma8xd-mba8xx", "tq,tqma8xd", "fsl,imx8dx";
 };
 
 &fec2 {
diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8qxp-mba8xx.dts b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mba8xx.dts
similarity index 97%
rename from arch/arm/dts/fsl-imx8qxp-tqma8qxp-mba8xx.dts
rename to arch/arm/dts/fsl-imx8qxp-tqma8xqp-mba8xx.dts
index 9f1a14118b..6eb23c75ff 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8qxp-mba8xx.dts
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mba8xx.dts
@@ -14,12 +14,12 @@
 
 /dts-v1/;
 
-#include "fsl-imx8qxp-tqma8qxp.dtsi"
+#include "fsl-imx8qxp-tqma8xqp.dtsi"
 #include <dt-bindings/net/ti-dp83867.h>
 
 / {
-	model = "TQ Systems i.MX8QXP TQMa8xX on MBa8xX";
-	compatible = "tq,tqma8qxp-mba8xx", "tq,tqma8qxp", "fsl,imx8qxp";
+	model = "TQ Systems i.MX8QXP TQMa8XQP on MBa8Xx";
+	compatible = "tq,tqma8xqp-mba8xx", "tq,tqma8xqp", "fsl,imx8qxp";
 
 	chosen {
 		bootargs = "console=ttyLP1,115200 earlycon=lpuart32,0x5a070000,115200";
diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8qxp.dtsi b/arch/arm/dts/fsl-imx8qxp-tqma8xqp.dtsi
similarity index 98%
rename from arch/arm/dts/fsl-imx8qxp-tqma8qxp.dtsi
rename to arch/arm/dts/fsl-imx8qxp-tqma8xqp.dtsi
index 3e231b9964..1ecd96acfb 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8qxp.dtsi
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqp.dtsi
@@ -19,8 +19,8 @@
 #include "fsl-imx8qxp.dtsi"
 
 / {
-	model = "TQ Systems i.MX8QXP TQMa8xX";
-	compatible = "tq,tqma8qxp", "fsl,imx8qxp";
+	model = "TQ Systems i.MX8QXP TQMa8XQP";
+	compatible = "tq,tqma8xqp", "fsl,imx8qxp";
 };
 
 &iomuxc {
diff --git a/arch/arm/mach-imx/imx8/Kconfig b/arch/arm/mach-imx/imx8/Kconfig
index aac0dc67f4..045397e9e2 100644
--- a/arch/arm/mach-imx/imx8/Kconfig
+++ b/arch/arm/mach-imx/imx8/Kconfig
@@ -77,8 +77,8 @@ config TARGET_IMX8QXP_MEK
 	select SUPPORT_SPL
 	select IMX8QXP
 
-config TARGET_TQMA8QX
-	bool "Support TQMa8xX board"
+config TARGET_TQMA8XX
+	bool "Support TQMa8Xx board"
 	select BOARD_LATE_INIT
 	select IMX8QXP
 	select ARCH_MISC_INIT
@@ -140,7 +140,7 @@ source "board/freescale/imx8qm_arm2/Kconfig"
 source "board/freescale/imx8qm_mek/Kconfig"
 source "board/freescale/imx8qxp_arm2/Kconfig"
 source "board/freescale/imx8qxp_mek/Kconfig"
-source "board/tqc/tqma8qx/Kconfig"
+source "board/tqc/tqma8xx/Kconfig"
 source "board/tqc/tqma8qxs/Kconfig"
 
 endif
diff --git a/board/tqc/tqma8qx/Kconfig b/board/tqc/tqma8qx/Kconfig
deleted file mode 100644
index 9034af89b7..0000000000
--- a/board/tqc/tqma8qx/Kconfig
+++ /dev/null
@@ -1,96 +0,0 @@
-if TARGET_TQMA8QX
-
-config SYS_BOARD
-	default "tqma8qx"
-
-config SYS_VENDOR
-	default "tqc"
-
-config SYS_CONFIG_NAME
-	default "tqma8qx"
-
-choice
-	prompt "TQMa8QX CPU variant"
-	default TQMA8QX_MX8QXP
-	help
-	  Select cpu variant for TQMa8QX
-
-config TQMA8QX_MX8QXP
-	bool "TQMa8QX with i.MX8QXP"
-	help
-	  Select the CPU i.MX8QXP / i.MX8DXP
-
-config TQMA8QX_MX8DX
-	bool "TQMa8QX with i.MX8QXP"
-	help
-	  Select the CPU i.MX8QXP / i.MX8DXP
-
-endchoice
-
-choice
-	prompt "TQMa8QX RAM size"
-	default TQMA8QX_RAM_1014MB if TQMA8QX_MX8QXP
-	default TQMA8QX_RAM_512MB if TQMA8QX_MX8DX
-	help
-	  Select RAM size for TQMa8QX
-
-config TQMA8QX_RAM_512MB
-	bool "TQMa8QX with 512 MiB RAM"
-	help
-	  Select variant with 512 MiB RAM
-
-config TQMA8QX_RAM_1024MB
-	bool "TQMa8QX with 1024 MiB RAM"
-	help
-	  Select variant with 1024 MiB RAM
-
-config TQMA8QX_RAM_2048MB
-	bool "TQMa8QX with 2048 MiB RAM"
-	help
-	  Select variant with 2048 MiB RAM
-
-endchoice
-
-choice
-	prompt "TQMa8QX base board variant"
-	default TQMA8QX_MBA8XX
-	help
-	  Select base board for TQMa8QX
-
-config TQMA8QX_MBA8XX
-	bool "TQMa8QX on MBa8xX Starterkit"
-	select TQC_SDMMC
-	select DM_ETH
-	help
-	  Select the MBa8QX starterkit. This features a GigE Phy, USB, SD-Card
-	  etc.
-
-endchoice
-
-config DEFAULT_DEVICE_TREE
-	default "fsl-imx8dx-tqma8dx-mba8xx" if TQMA8QX_MX8DX && TQMA8QX_MBA8XX
-	default "fsl-imx8qxp-tqma8qxp-mba8xx" if TQMA8QX_MX8QXP && TQMA8QX_MBA8XX
-
-config DEFAULT_FDT_FILE
-	default "fsl-imx8dx-tqma8dx-mba8xx.dtb" if TQMA8QX_MX8DX && TQMA8QX_MBA8XX
-	default "fsl-imx8qxp-tqma8qxp-mba8xx.dtb" if TQMA8QX_MX8QXP && TQMA8QX_MBA8XX
-
-choice
-	prompt "TQMa8QX boot device"
-	default TQMBA8QX_SDCARD
-	help
-	  Select boot device for TQMa8QX
-
-config TQMBA8QX_SDCARD
-	bool "TQMa8QX from SD / e-MMC"
-	select SD_BOOT
-	select ENV_IS_IN_MMC
-	help
-	  Select booting from SD / e-MMC
-
-endchoice
-
-
-source "board/tqc/common/Kconfig"
-
-endif
diff --git a/board/tqc/tqma8xx/Kconfig b/board/tqc/tqma8xx/Kconfig
new file mode 100644
index 0000000000..e54901cf0b
--- /dev/null
+++ b/board/tqc/tqma8xx/Kconfig
@@ -0,0 +1,96 @@
+if TARGET_TQMA8XX
+
+config SYS_BOARD
+	default "tqma8xx"
+
+config SYS_VENDOR
+	default "tqc"
+
+config SYS_CONFIG_NAME
+	default "tqma8xx"
+
+choice
+	prompt "TQMa8Xx CPU variant"
+	default TQMA8XX_MX8QXP
+	help
+	  Select cpu variant for TQMa8Xx
+
+config TQMA8XX_CPU_MX8QXP
+	bool "TQMa8Xx with i.MX8QXP / i.MX8DXP"
+	help
+	  Select CPU i.MX8QXP / i.MX8DXP
+
+config TQMA8XX_CPU_MX8DX
+	bool "TQMa8Xx with i.MX8DX"
+	help
+	  Select CPU i.MX8DX
+
+endchoice
+
+choice
+	prompt "TQMa8Xx RAM size"
+	default TQMA8XX_RAM_1024MB if TQMA8XX_CPU_MX8QXP
+	default TQMA8XX_RAM_512MB if TQMA8XX_CPU_MX8DX
+	help
+	  Select RAM size for TQMa8Xx
+
+config TQMA8XX_RAM_512MB
+	bool "TQMa8Xx with 512 MiB RAM"
+	help
+	  Select variant with 512 MiB RAM
+
+config TQMA8XX_RAM_1024MB
+	bool "TQMa8Xx with 1024 MiB RAM"
+	help
+	  Select variant with 1024 MiB RAM
+
+config TQMA8XX_RAM_2048MB
+	bool "TQMa8Xx with 2048 MiB RAM"
+	help
+	  Select variant with 2048 MiB RAM
+
+endchoice
+
+choice
+	prompt "TQMa8Xx base board variant"
+	default TQMA8XX_BB_MBA8XX
+	help
+	  Select base board for TQMa8Xx
+
+config TQMA8XX_BB_MBA8XX
+	bool "TQMa8Xx on MBa8xX Starterkit"
+	select TQC_SDMMC
+	select DM_ETH
+	help
+	  Select the MBa8Xx starterkit. This features a GigE Phy, USB, SD-Card
+	  etc.
+
+endchoice
+
+config DEFAULT_DEVICE_TREE
+	default "fsl-imx8dx-tqma8xd-mba8xx" if TQMA8XX_CPU_MX8DX && TQMA8XX_BB_MBA8XX
+	default "fsl-imx8qxp-tqma8xqp-mba8xx" if TQMA8XX_CPU_MX8QXP && TQMA8XX_BB_MBA8XX
+
+config DEFAULT_FDT_FILE
+	default "fsl-imx8dx-tqma8xd-mba8xx.dtb" if TQMA8XX_CPU_MX8DX && TQMA8XX_BB_MBA8XX
+	default "fsl-imx8qxp-tqma8xqp-mba8xx.dtb" if TQMA8XX_CPU_MX8QXP && TQMA8XX_BB_MBA8XX
+
+choice
+	prompt "TQMa8Xx boot device"
+	default TQMA8XX_BOOT_SDCARD
+	help
+	  Select boot device for TQMa8Xx
+
+config TQMA8XX_BOOT_SDCARD
+	bool "TQMa8Xx from SD / e-MMC"
+	select SD_BOOT
+	select ENV_IS_IN_MMC
+	help
+	  Select booting from SD / e-MMC
+
+endchoice
+
+
+source "board/tqc/common/Kconfig"
+
+endif
diff --git a/board/tqc/tqma8qx/Makefile b/board/tqc/tqma8xx/Makefile
similarity index 51%
rename from board/tqc/tqma8qx/Makefile
rename to board/tqc/tqma8xx/Makefile
index 43f8cd935c..8051e72da4 100644
--- a/board/tqc/tqma8qx/Makefile
+++ b/board/tqc/tqma8xx/Makefile
@@ -4,6 +4,5 @@
 # SPDX-License-Identifier:	GPL-2.0+
 #
 
-obj-y += tqma8qx.o
-obj-$(CONFIG_TQMA8QX_MBA8XX) += tqma8qx-mba8xx.o
-
+obj-y += tqma8xx.o
+obj-$(CONFIG_TQMA8XX_BB_MBA8XX) += tqma8xx-mba8xx.o
diff --git a/board/tqc/tqma8qx/tqma8qx-mba8xx.c b/board/tqc/tqma8xx/tqma8xx-mba8xx.c
similarity index 99%
rename from board/tqc/tqma8qx/tqma8qx-mba8xx.c
rename to board/tqc/tqma8xx/tqma8xx-mba8xx.c
index bcb47cf407..44ddcc44ce 100644
--- a/board/tqc/tqma8qx/tqma8qx-mba8xx.c
+++ b/board/tqc/tqma8xx/tqma8xx-mba8xx.c
@@ -131,6 +131,8 @@ int tqc_bb_board_late_init(void)
 #ifdef CONFIG_ENV_IS_IN_MMC
 	board_late_mmc_env_init();
 #endif
+
+	return 0;
 }
 
 void board_quiesce_devices()
diff --git a/board/tqc/tqma8qx/tqma8qx.c b/board/tqc/tqma8xx/tqma8xx.c
similarity index 100%
rename from board/tqc/tqma8qx/tqma8qx.c
rename to board/tqc/tqma8xx/tqma8xx.c
diff --git a/configs/tqma8dx_mba8xx_mmc_defconfig b/configs/tqma8xd_mba8xx_mmc_defconfig
similarity index 96%
rename from configs/tqma8dx_mba8xx_mmc_defconfig
rename to configs/tqma8xd_mba8xx_mmc_defconfig
index 4b711f4a85..5904298d6f 100644
--- a/configs/tqma8dx_mba8xx_mmc_defconfig
+++ b/configs/tqma8xd_mba8xx_mmc_defconfig
@@ -2,8 +2,8 @@ CONFIG_ARM=y
 CONFIG_ARCH_IMX8=y
 CONFIG_SYS_TEXT_BASE=0x80020000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
-CONFIG_TARGET_TQMA8QX=y
-CONFIG_TQMA8QX_MX8DX=y
+CONFIG_TARGET_TQMA8XX=y
+CONFIG_TQMA8XX_CPU_MX8DX=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTDELAY=3
diff --git a/configs/tqma8qx_mba8qx_mmc_defconfig b/configs/tqma8xqp_mba8xx_mmc_defconfig
similarity index 98%
rename from configs/tqma8qx_mba8qx_mmc_defconfig
rename to configs/tqma8xqp_mba8xx_mmc_defconfig
index a27cd29642..2e95a93506 100644
--- a/configs/tqma8qx_mba8qx_mmc_defconfig
+++ b/configs/tqma8xqp_mba8xx_mmc_defconfig
@@ -2,7 +2,7 @@ CONFIG_ARM=y
 CONFIG_ARCH_IMX8=y
 CONFIG_SYS_TEXT_BASE=0x80020000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
-CONFIG_TARGET_TQMA8QX=y
+CONFIG_TARGET_TQMA8XX=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTDELAY=3
diff --git a/include/configs/tqma8qx-mba8xx.h b/include/configs/tqma8xx-mba8xx.h
similarity index 86%
rename from include/configs/tqma8qx-mba8xx.h
rename to include/configs/tqma8xx-mba8xx.h
index 7e1e026fc5..ddf6549593 100644
--- a/include/configs/tqma8qx-mba8xx.h
+++ b/include/configs/tqma8xx-mba8xx.h
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier:	GPL-2.0+
  */
 
-#if !defined(__TQMA8QX_MBA8QX_H)
-#define __TQMA8QX_MBA8QX_H
+#if !defined(__TQMA8XX_MBA8XX_H)
+#define __TQMA8XX_MBA8XX_H
 
 #define CONFIG_FEC_MXC
 #define CONFIG_FEC_XCV_TYPE		RGMII
@@ -26,4 +26,4 @@
 	"console=ttyLP1,115200 earlycon=lpuart32,5a070000,115200\0" \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0"
 
-#endif /* __TQMA8QX_MBA8XX_H */
+#endif /* __TQMA8XX_MBA8XX_H */
diff --git a/include/configs/tqma8qx.h b/include/configs/tqma8xx.h
similarity index 96%
rename from include/configs/tqma8qx.h
rename to include/configs/tqma8xx.h
index 93d0a01f9f..02b35a0e6d 100644
--- a/include/configs/tqma8qx.h
+++ b/include/configs/tqma8xx.h
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier:	GPL-2.0+
  */
 
-#ifndef __TQMA8QX_H
-#define __TQMA8QX_H
+#ifndef __TQMA8XX_H
+#define __TQMA8XX_H
 
 #include <linux/sizes.h>
 #include <asm/arch/imx-regs.h>
@@ -114,7 +114,7 @@
             "\0" \
 
 /* Initial environment variables */
-#define TQMA8QX_MODULE_ENV_SETTINGS		\
+#define TQMA8XX_MODULE_ENV_SETTINGS		\
 	CONFIG_MFG_ENV_SETTINGS \
 	M4_BOOT_ENV \
 	XEN_BOOT_ENV \
@@ -308,11 +308,11 @@
 #define CONFIG_NR_DRAM_BANKS		1
 #define PHYS_SDRAM_1			0x80000000
 
-#if defined(CONFIG_TQMA8QX_RAM_512MB)
+#if defined(CONFIG_TQMA8XX_RAM_512MB)
 #define PHYS_SDRAM_1_SIZE		SZ_512M
-#elif defined(CONFIG_TQMA8QX_RAM_1014MB)
+#elif defined(CONFIG_TQMA8XX_RAM_1024MB)
 #define PHYS_SDRAM_1_SIZE		SZ_1G
-#elif defined(CONFIG_TQMA8QX_RAM_2048MB)
+#elif defined(CONFIG_TQMA8XX_RAM_2048MB)
 #define PHYS_SDRAM_1_SIZE		SZ_2G
 #else
 #error
@@ -373,24 +373,24 @@
 #define CONFIG_SYS_MEMTEST_SCRATCH CONFIG_SYS_MEMTEST_END
 #endif
 
-#if defined(CONFIG_TQMA8QX_MX8QXP)
-#define TQMA8_BOARD_NAME	"TQMa8QXP"
+#if defined(CONFIG_TQMA8XX_CPU_MX8QXP)
+#define TQMA8_BOARD_NAME	"TQMa8XQP"
 #define TQMA8_BOARD_REV		"iMX8QXP"
-#elif defined(CONFIG_TQMA8QX_MX8DX)
-#define TQMA8_BOARD_NAME	"TQMa8DX"
+#elif defined(CONFIG_TQMA8XX_CPU_MX8DX)
+#define TQMA8_BOARD_NAME	"TQMa8XD"
 #define TQMA8_BOARD_REV		"iMX8DX"
 #else
 #error
 #endif
 
-#if defined(CONFIG_TQMA8QX_MBA8XX)
-#include "tqma8qx-mba8xx.h"
+#if defined(CONFIG_TQMA8XX_BB_MBA8XX)
+#include "tqma8xx-mba8xx.h"
 #else
 #error
 #endif
 
 #define CONFIG_EXTRA_ENV_SETTINGS		\
-	TQMA8QX_MODULE_ENV_SETTINGS		\
+	TQMA8XX_MODULE_ENV_SETTINGS		\
 	BB_ENV_SETTINGS
 
 #ifndef CONFIG_SPL_BUILD
@@ -400,4 +400,4 @@
 #include <config_distro_bootcmd.h>
 #endif
 
-#endif /* __TQMA8QX_H */
+#endif /* __TQMA8X_H */
